internal/storage/storage.go

```diff
@@ -381,18 +382,19 @@ func (s *UnstructuredObjectStore) Put(path string, object map[string]any, raw []
 
 	storeObject := StoreObject{Path: path, Unstructured: u, Hash: NewSHA256(raw)}
 
+	var err error
 	index := GetResourceIndex(storeObject)
 	if _, ok := s.Storage[index]; ok {
 		// for cert-manager migration we have duplicated resources for legacy version
 		// it's ok for cluster but is not expected by tests. Remove it after legacy version will be removed
 		if strings.Contains(index.AsString(), "ClusterIssuer") || strings.HasPrefix(index.AsString(), "d8-cert-manager") {
 			return nil
 		}
-		return fmt.Errorf("object %q already exists in the object store", index.AsString())
+		err = errors.Join(err, fmt.Errorf("object %q already exists", index.AsString()))
```

Comment:
can't find second error to use join
```suggestion
		err = fmt.Errorf("object %q already exists", index.AsString())
```